src/cms/static/js/DeleteRowTable.js

```diff
@@ -0,0 +1,4 @@
+var x = document.getElementsByTagName("tr");
+for(i=5; i<= x.length; i++){
+    document.getElementById("myTable").deleteRow(5);
+}
```

Comment:
This is bad practise. Please select the 5 most recent feedback entries in the python view and pass only them to the template, so you don't have to perform any client-side dom-modification to filter them. You can perform `LIMIT`-database queries with Python list selectors (`list[:5]`), see examples in the [Django documentation](https://docs.djangoproject.com/en/2.2/topics/db/queries/#limiting-querysets).